snapcraft/tests/unit/sources/test_rpm.py

```diff
@@ -42,6 +42,10 @@ def test_pull_rpm_file_must_extract(self):
         rpm_source = sources.Rpm(rpm_file_path, dest_dir)
         rpm_source.pull()
 
+        
+        rpm_source2 = sources.Rpm(rpm_file_path, dest_dir,2)
```

Comment:
Can you please add a separate method for this? Maybe `test_pull_rpm_with_source_tag`.

And I think using the argument name would make this more readable, eg. `source_tag='foo'`.